packages/react-console/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "dependencies": {
     "@novnc/novnc": "^1.3.0",
-    "@patternfly/patternfly": "4.222.4",
+    "@patternfly/patternfly": "4.223.2",
```

Comment:
can we bump to [4.224.0](https://www.npmjs.com/package/@patternfly/patternfly/v/4.224.0) in this PR? hopefully that's the last core version for this release, it includes support for this v4 issue https://github.com/patternfly/patternfly-react/issues/8434